libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -1216,6 +1217,20 @@ bool AP_Arming::visodom_checks(bool display_failure) const
     return true;
 }
 
+// check disarm switch is asserted
+bool AP_Arming::disarm_switch_checks(bool display_failure) const
+{
+    const RC_Channel *chan = rc().find_channel_for_option(RC_Channel::AUX_FUNC::DISARM);
+    if (chan != nullptr &&
+            chan->get_aux_switch_pos() == RC_Channel::aux_switch_pos_t::HIGH &&
```

Comment:
Whitepsace issues; indented incorrectly.

Reply:
fixed it